### ISSUE TITLE
Metrics being collected with gaps between each time period

### ISSUE THREAD

**Issue by gabricc** [sj26/sidekiq-cloudwatchmetrics#19]
Hi everyone!

I added the **sidekiq-cloudwatchmetrics** gem to our code and followed the instructions you provided in the Readme. The metrics started to appear on CloudWatch, but there are gaps between each time period, which doesn’t seem to be normal... Ex:

![image](https://user-images.githubusercontent.com/42014588/161141230-f26e1d8f-65d5-4d11-976f-2aee48c3cee7.png)

If you see in the image above, we have metrics at 13:27h, between 13:45 to 14h and around 14:34. Aside from these time periods, we don’t have the metrics.

Our app is running on an ECS Fargate Cluster, this is the sidekiq.rb file:
```
require "sidekiq"
require "sidekiq/cloudwatchmetrics"

Sidekiq::CloudWatchMetrics.enable!
```

Maybe there is something missing in my config, can you help with that? Thanks!

**Comment by sj26** [comment]
@gabricc this might be related to #16. Do you see anything in your sidekiq logs? Other folks are seeing messages like this:

```
Aws::CloudWatch::Errors::InvalidParameterValue: The value NaN for parameter MetricData.member.11.Value is invalid.
```

**Comment by gabricc** [comment]
Hi @sj26 ! This is exactly the error I was getting, can you tell me if it was fixed?

**Comment by sj26** [comment]
Thanks, this should be fixed in [version 2.4.0](https://github.com/sj26/sidekiq-cloudwatchmetrics/releases/tag/v2.4.0).
